pkg/downloader/chart_downloader.go

```diff
@@ -97,6 +98,13 @@ func atomicWriteFile(filename string, body io.Reader, mode os.FileMode) error {
 	return os.Rename(tempName, filename)
 }
 
+type InMemoryChart struct {
+	Chart                  *bytes.Buffer
```

Comment:
I don't believe we need to make this type public.  So it should start with a lowercase letter instead.

Reply:
Done